plugins/multiarc/custom.cpp

```diff
@@ -989,9 +989,9 @@ void ParseListingItemRegExp(Match match,
 {
 
     if(const char *p = match["name"])
-        lstrcat(Item->FindData.cFileName, p);
+        lstrcpyn(Item->FindData.cFileName, p, sizeof(Item->FindData.cFileName));
```

Comment:
Why the original code was appending the strings rather than copying them?

Reply:
Item - this structure must be filled:
https://github.com/FarGroup/FarManager/blob/86c26a80a21e24e3c2a7b4d259a76d3ef34d44fc/plugins/multiarc/README.TXT#L65

Reply:
I think it always should be empty
https://github.com/FarGroup/FarManager/blob/86c26a80a21e24e3c2a7b4d259a76d3ef34d44fc/plugins/multiarc/arcread.cpp#L97